addon/components/fd-visual-edit-control-tree.js

```diff
@@ -180,10 +180,10 @@ export default Ember.Component.extend({
       attribute = aggregation.findBy('endRole', namesPropertyDefinition[0]);
     } else if (propertyDefinition instanceof FdViewAttributesMaster) {
       this.set('selectedItem.type', 'master');
-      let parsingResult = this._parsingPropertyName(namesPropertyDefinition);
+      let parsingResult = parsingPropertyName(this.get('store'), this.get('model.dataobject'), namesPropertyDefinition);
```

Comment:
`this.get('model.dataobject')` - are you sure that this is here?

Reply:
yes